scripts/news/utils.py

```diff
@@ -0,0 +1,250 @@
+import glob
+import itertools
+import tempfile
+import textwrap
+from datetime import datetime, timezone
+from typing import Any, Dict, Generator, List, Mapping, Optional, Tuple, Union
+
+import click
+import requests
+import tomli
+from click import Context, Option, echo, style
+from jinja2 import Template
+
+from .constants import *
```

Comment:
please don't use star imports. 

Two solutions are defining everything that we import, or just import constants itself.

Reply:
Using `constants.` everywhere is painful and importing everything is even more painful, I dunno what else to do

Reply:
you must do one of the two.